src/opensubtitles.html

```diff
@@ -0,0 +1,28 @@
+﻿<div id="openSubtitlesPage" data-role="page" class="page type-interior metadataConfigurationPage" data-require="scripts/opensubtitles,emby-input,emby-button,emby-linkbutton">
+    <div>
+        <div class="content-primary">
+            <form class="metadataSubtitlesForm">
+                <div style="height:0; overflow: hidden;"><input type="text" name="fakeusernameremembered" tabindex="-1" /><input type="password" name="fakepasswordremembered" tabindex="-1" /></div>
+
+                <div style="display: none;">
+                    <input type="text" id="PreventChromeAutocomplete" name="PreventChromeAutocomplete" autocomplete="address-level4" />
+                </div>
+                <br />
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtOpenSubtitleUsername" autocomplete="off" label="${LabelOpenSubtitlesUsername}" />
+                    <div class="fieldDescription">
+                        <!-- TODO missing ${ButtonRegister} -->
+                        <a is="emby-linkbutton" class="button-link" target="_blank" href="http://www.opensubtitles.org/">Register</a>
```

Comment:
Why it's an `<a>`, not a `<button>` like other "Save" button below?

Reply:
Anywhere with a link just loads a separate page with no other background tasks, while buttons are for API calls or other stuff besides the page load.

Reply:
Thank you for explanation.
I feel this should be somewhere in UX docs... cc @nvllsvm

Reply:
This isn't necessarily a coding style, just something I noticed in the codebase. It makes sense to use a simple link when you just need to load a separate page.

Reply:
Not a coding, but a UX style. So our user experience is consistent :)